packages/core/src/js/feedback/integration.ts

```diff
@@ -0,0 +1,31 @@
+import type { Integration } from '@sentry/core';
+
+import { defaultConfiguration } from './defaults';
+import defaultStyles from './FeedbackForm.styles';
+import type { FeedbackFormProps } from './FeedbackForm.types';
+
+export const FEEDBACK_FORM_INTEGRATION_NAME = 'MobileFeedback';
+
+type FeedbackIntegration = Integration & {
+  options: Partial<FeedbackFormProps>;
+};
+
+let savedOptions: Partial<FeedbackFormProps> = {};
+
+export const feedbackIntegration = (initOptions: FeedbackFormProps = {}): FeedbackIntegration => {
+  savedOptions = {
+    ...defaultConfiguration,
+    ...initOptions,
+    styles: {
+      ...defaultStyles,
+      ...initOptions.styles,
```

Comment:
m: We can save the `initOptions` as is, the `FeedbackForm` will merge in the defaults.

Reply:
Good point @krystofwoldrich 👍 
Updated with cbb1fd198864d777027efca9d8c9d87e4163dacc and added a test.